static/js/portico/landing-page.js

```diff
@@ -68,23 +48,24 @@ const apps_events = function () {
             alt: "Android",
             description: "Zulip's native Android app makes it easy to keep up while on the go.",
             show_instructions: false,
-            link: "https://play.google.com/store/apps/details?id=com.zulipmobile",
+            play_store_link: "https://play.google.com/store/apps/details?id=com.zulipmobile",
+            download_link: "/apps/download/android",
```

Comment:
We don’t need this for Android because https://github.com/zulip/zulip-mobile/releases/latest/download/app-release.apk already works.